src/Symfony/Component/Validator/Constraints/Range.php

```diff
@@ -46,6 +46,17 @@ class Range extends Constraint
     public $max;
     public $maxPropertyPath;
 
+    // BC layer, to remove in 6.0
```

Comment:
why not 5.0?

Reply:
Indeed the new behavior is already present in 5.0.0, changed.